t/correct_pod.t

```diff
@@ -0,0 +1,21 @@
+use warnings;
```

Comment:
strict first, then warnings - for some reason all these have them swapped?

Reply:
my code editor changes the order. I'll fix it.